api/Cargo.toml

```diff
@@ -10,7 +10,6 @@ chrono = "0.4.34"
 reqwest = { version = "0.11.24", features = ["blocking", "json"] }
 serde = "1.0.196"
 serde_json = "1.0.113"
-snailquote = "0.3.1"
```

Comment:
failed experiment that was the root cause of the rendering bug, this removed the double quotes from the strings and resulted in invalid json string, so serde_json failed to parse.